camera_frame_id: usb_cam
pos_init:
- [0.0]
- [0.0]
- [0.0]
world_map:
- [0.752, -2.23, 0.0, 0.0]
- [0.5583, 1.1799, 1.5708, 1.0]
- [1.3703, 1.1799, 1.5708, 2.0]
- [1.8778, 0.9516, 0.0, 3.0]
- [-4.30, 4.06, 0.0, 4.0]
- [-2.41, 4.28, 0.0, 5.0]
- [-0.129, 1.23, 0.0, 6.0]
- [-0.043, -2.4, 0.0, 7.0]
- [1.8778, 0.6471, 0.0, 8.0]
- [0.2538, 1.1799, 1.5708, 9.0]
